packages/sit-onyx/src/styles/z-indices.scss

```
@@ -0,0 +1,10 @@
+// all z-indices that are needed in the onyx design system
+:root {
+  --onyx-z-index-overlay: 70;
+  --onyx-z-index-nav-bar: 60;
```

```suggestion
  --onyx-z-index-navigation: 60;
```

This layer would be used for the header and sidebar, right? So its not only for the nav bar